app/src/components/v-select/v-select.vue

```diff
@@ -132,7 +132,7 @@ function useItems() {
 				selectable: get(item, props.itemSelectable),
 				children: children
 					? children.filter((childItem: Record<string, any>) =>
-							filterItem(get(childItem, props.itemText), get(childItem, props.itemValue), childItem.children),
```

Comment:
This is an unrelated fix. `children` at this point is already parsed and transformed to a canonical item form, which now only has the `text` and `value` props, and not the custom ones that one could provide through `itemText` and `itemValue`.